projects/kit/components/pagination/pagination.template.html

```diff
@@ -62,6 +62,7 @@
             type="button"
             class="t-button t-button_small"
             [appearance]="getElementMode(indexItem)"
+            [attr.title]="carousel()?.index"
```

Comment:
Let's not add this here. Title is the last resort and we should only use it in demo if we can, because there's no way to style it. In this case these are dots for each item to click and show it. I think we can just put `indexItem  + 1` in button's content and make font-size: 0 here, no need to expand i18n object for it. We should only expand i18n if there's no way around that.